projects/Rockchip/devices/RK3328/linux/default/linux.aarch64.conf

```diff
@@ -6245,7 +6247,6 @@ CONFIG_CRYPTO_CRC32C=y
 CONFIG_CRYPTO_CRC32=y
 CONFIG_CRYPTO_XXHASH=m
 CONFIG_CRYPTO_BLAKE2B=m
-CONFIG_CRYPTO_BLAKE2S=m
```

Comment:
I don't know if it's useful, but there is a ``CONFIG_CRYPTO_BLAKE2S_ARM`` module now, which was recently [enabled on Debian](https://salsa.debian.org/kernel-team/linux/-/commit/2388a96aa63c1f06d1630544da37c8cfd1b484fc)

Reply:
Hi @diederikdehaas _GENERIC is used on arm64, as there is no arch specific BLAKE2 for arm64 (there is for x86 and arm)
- https://github.com/torvalds/linux/tree/master/arch/arm64/crypto
- https://github.com/torvalds/linux/tree/master/arch/arm/crypto
- https://github.com/torvalds/linux/tree/master/arch/x86/crypto

the Debian reference will be for 32-bit kernels.

```
$ git grep CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S
projects/Allwinner/linux/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/Generic/linux/linux.x86_64.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/NXP/devices/iMX6/linux/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/RPi/devices/RPi/linux/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/RPi/devices/RPi2/linux/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/Rockchip/devices/RK3288/linux/default/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
projects/Samsung/linux/linux.arm.conf:CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=y
```

Reply:
Oh, you're entirely right :+1: Thanks for clearing that up :-)